utilities/node_activator/README.md

```diff
@@ -6,7 +6,29 @@ A module to activate VM nodes.
 
 ## Installation
 
-TBD
+In order to use `NodeActivator`, you will need Elixir installed. Then create an Elixir project, whose name is `my_app`, for example, via the `mix` build tool:
+
+```sh
+$ mix new my_app
+```
+
+Then you can add `NodeActivator` as dependency in your `mix.exs`:
```

Comment:
I don't know what is best. I am not a native English speaker 😅 

"as a dependency" and "add ... to ..." sound better to my ears.

```suggestion
Then you can add `NodeActivator` as a dependency to your `mix.exs`:
```

```suggestion
Then you can add `NodeActivator` to the dependencies in your `mix.exs`:
```


```suggestion
Then you can add `NodeActivator` to the list of dependencies in your `mix.exs`:
```